### ISSUE TITLE
[Request] Add New Features.

### ISSUE THREAD

**Issue by jr0driguezv** [Treeki/FlicksForAll#5]
Hi,
I like how Tweak's development is going, but it would be excellent if I could add the following functionality:

- **SwipeUp:** To enter the character. (It is more understandable in this way, since the character is at the top of each Keyboard Letter)

- **SwipeDown:** To enter the letter in Uppercase

**Do you think it would be possible to add these functionalities?**

These features are from [SwipeExpander](https://moreinfo.thebigboss.org/moreinfo/depiction.php?file=swipeexpanderDp), but this Tweak has not been updated since iOS 9 and Crash when you open the Notes application.

**Comment by pmbonea** [comment]
Swipe up could be more useful than swipe down…as sometimes it activate reachability

Thanks for this amazing tweak!

**Comment by jr0driguezv** [comment]
@pmbonea  Correct, it is often annoying when reachability is activated. But @Treeki  haven't updated the Tweak again ...

**Comment by jebediah2** [comment]
Swipe up would take all former SwipeExpander users on board, me included.
Swipe down for uppercase also a good option though not as important.
